.github/workflows/ci.yml

```diff
@@ -18,4 +18,4 @@ concurrency:
 jobs:
   puppet:
     name: Puppet
-    uses: voxpupuli/gha-puppet/.github/workflows/basic.yml@v1
+    uses: voxpupuli/gha-puppet/.github/workflows/beaker.yml@v2
```

Comment:
I think we should explicitly disable acceptance tests in `sync.yml`, at least for now.